topi/python/topi/cuda/pooling.py

```diff
@@ -84,6 +87,23 @@ def traverse(OP):
     traverse(outs[0].op)
     return s
 
+@generic.schedule_global_pool.register(["cuda", "gpu"])
+def schedule_global_pool(outs):
```

Comment:
Should we just remove `schedule_global_pool` since `global_pool` now just uses `adaptive_pool`?